.gitignore

```diff
@@ -0,0 +1,11 @@
+## IDE Specific
+.idea/
+out/s
+*.iml
+*.ipr
+*.iws
+
+.settings/
+.classpath
+.project
+s
```

Comment:
@takezoe **every** Git repo should have ```.gitignore``` for various temporary files too, not just IDE.